src/encoded/types/biosample.py

```diff
@@ -59,18 +73,54 @@ class Biosample(Item):  # CalculatedBiosampleSlims, CalculatedBiosampleSynonyms)
         'cell_culture_details.morphology_image.attachment.download',
         'cell_culture_details.morphology_image.attachment.width',
         'cell_culture_details.morphology_image.attachment.height',
+
+        # OntologyTerm linkTo
         'cell_culture_details.tissue.term_name',
         'cell_culture_details.tissue.preferred_name',
+        'cell_culture_details.tissue.slim_terms',
+        'cell_culture_details.tissue.synonyms',
+        'cell_culture_details.tissue.term_id',
+
+        # Modification linkTo
         'modifications.modification_type',
+        'modifications.genomic_change',
+        'modifications.target_of_mod',
+        'modifications.override_modification_name',
         'modifications.description',
+
+        # Treatment linkTo, verbose display_title so many embeds necessary
         'treatments.treatment_type',
         'treatments.description',
+        'treatments.chemical',
+        'treatments.biological_agent',
+        'treatments.constructs',
+        'treatments.duration',
+        'treatments.duration_units',
+        'treatments.concentration',
+        'treatments.concentration_units',
+        'treatments.temperature',
+
+        # Construct linkTo
+        'treatments.constructs.name',
+
+        # Biofeature linkTo
+        'treatments.constructs.expression_products.feature_type',
+        'treatments.constructs.expression_products.preferred_label',
+        'treatments.constructs.expression_products.cellular_structure',
+        'treatments.constructs.expression_products.organism_name',
         'treatments.constructs.expression_products.relevant_genes',
-        'treatments.constructs.expression_products.feature_mods.mod_type',
+        'treatments.constructs.expression_products.feature_mods',
```

Comment:
`treatments.constructs.expression_products.feature_mods.mod_position` (I see you already have the other feature_mods prop below)